be/src/runtime/current_thread.h

```diff
@@ -55,6 +55,7 @@ class CurrentThread {
         _fragment_instance_id = fragment_instance_id;
     }
     void set_pipeline_driver_id(int32_t driver_id) { _driver_id = driver_id; }
+    int32_t get_driver_id() const { return _driver_id; }
```

Comment:
Did we assign driver id in pip_scan_io thread ?

Reply:
Fixed. Share the same driver_id in io-thread.